cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -4,6 +4,8 @@ set -e
 
 export CLUSTER_NAME="sriov"
 
+SRIOV_TESTS_NS="${SRIOV_TESTS_NS:-kubevirt-test-default1}"
```

Comment:
I think you need to explain what `ubevirt-test-default1` is and from where it got here.
There is a dependency on the test code which is bad, we need to figure out how to solve it (perhaps pass in the name to the tests).

Reply:
We solved it in the past by creating the pod preset in kubevirt, not in e2e tests, there is a PR that does it

Reply:
https://github.com/kubevirt/kubevirt/pull/3374

Reply:
I continued this [here](https://github.com/kubevirt/kubevirtci/pull/570#issuecomment-801952386).

Reply:
@ormergi was this missed? I do not see an explanation from where this name came from. It is hard to understand without a small explanation.

Reply:
@EdDev  I updated the commit message with expiation about this variable and default value

Reply:
Done